ndk/src/lib.rs

```diff
@@ -26,5 +26,6 @@ pub mod looper;
 pub mod media;
 pub mod native_activity;
 pub mod native_window;
+#[cfg(feature = "api-level-28")]
```

Comment:
That doesn't quite follow the standard of the other modules, I think [I requested](https://github.com/rust-windowing/android-ndk-rs/pull/267#issuecomment-1134595941) a top-level `#![]` annotation _inside_ the `surface_texture` module.